test_suite/slice.sugar.jsonnet

```diff
@@ -0,0 +1,106 @@
+local arr = std.range(0, 5);
+local str = "012345";
+local a = 2;
+local b = 4;
+
+local arrCases = [
+    {
+        input:  arr[2:4],
+        output: [ 2, 3 ],
+    },
+    {
+        input:  arr[2:4:1],
+        output: [ 2, 3 ],
+    },
+    {
+        input:  arr[:4],
+        output: [ 0, 1, 2, 3 ],
+    },
+    {
+        input:  arr[2:],
+        output: [ 2, 3, 4, 5 ],
+    },
+    {
+        input:  arr[:],
+        output: arr,
+    },
+    {
+        input:  arr[:1000],
+        output: arr,
+    },
+    //{
+    //    input:  arr[::],
+    //    output: arr,
+    //},
```

Comment:
@sparkprime This is breaking with:
```
FAIL (exit code): slice.sugar.jsonnet
This run's output:
STATIC ERROR: slice.sugar.jsonnet:32:21-22: Not a unary operator: ::
```
I'll investigate what is going on but it's not immediately obvious to me.